### PR TITLE
Issue #87: Added VideoPlaybackQuality to list of at risk features

### PR DESCRIPTION
**Pull request by jdsmith3000**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -625,12 +625,12 @@ <h2 id="w3c-editor-s-draft-06-june-2016"><abbr title="World Wide Web Consortium"
...
         <p>Implementors should be aware that this specification is not stable. <strong>Implementors who are not taking part in the discussions are likely to find the specification changing out from under them in incompatible ways.</strong> Vendors interested in implementing this specification before it eventually reaches the Candidate Recommendation stage should join the mailing list mentioned below and take part in the discussions.</p>
 
-        <p>The following features are <strong>at risk</strong> and may be removed due to lack of implementation.
-        </p>
+        <p>The following features are <strong>at risk</strong> and may be removed:</p>
         <ul>
-          <li><code><a href="#widl-VideoPlaybackQuality-totalFrameDelay">totalFrameDelay</a></code></li>
-          <li><code><a href="#widl-SourceBuffer-appendStream-void-ReadableStream-stream-unsigned-long-long-maxSize">appendStream()</a></code></li>
-        </ul>
+          <li><p>The <code><a href="#widl-SourceBuffer-appendStream-void-ReadableStream-stream-unsigned-long-long-maxSize">appendStream</a></code> method on the <a href="#sourcebuffer">sourceBuffer</a> object and algorithm steps that use it.</p></li>
```
Comment (wolenetz) [inline index.html]: s/that use it/that are used only by it/

File: index.html [hunk 2]
```diff
@@ -3187,7 +3187,15 @@ <h3 id="h-sourcebufferlist-events" resource="#h-sourcebufferlist-events"><span p
         </thead>
         <tbody>
           <tr>
-            <td>06 June 2016</td>
+            <td>07 June 2016</td>
```
Comment (wolenetz) [inline index.html]: rebase to current gh-pages and regenerate please, so the other recent changes aren't lost.

**Comment by wolenetz** [comment]
Looks pretty good - just a couple comments (see above).

File: index.html (hunk 2 above)
Comment (jdsmith3000) [inline index.html]: I thought the "Squash and Merge" function removed the need for doing a separate rebase, and that all we needed to do was resolve merge conflicts before initiating the merge.  Should we also run "git rebase gh-pages" on the branch before we merge?

Comment (wolenetz) [inline index.html]: I usually use git rebase in my workflow. I'm less clear if merge does the necessary things right. The end result is what matters :)

**Pull request merged**